xbmc/utils/SeekHandler.cpp

```diff
@@ -298,9 +307,102 @@ bool CSeekHandler::OnAction(const CAction &action)
         Seek(action.GetID() == ACTION_ANALOG_SEEK_FORWARD, action.GetAmount(), action.GetRepeat(), true);
       return true;
     }
+    case REMOTE_0:
+    case REMOTE_1:
+    case REMOTE_2:
+    case REMOTE_3:
+    case REMOTE_4:
+    case REMOTE_5:
+    case REMOTE_6:
+    case REMOTE_7:
+    case REMOTE_8:
+    case REMOTE_9:
+    {
+      if (!g_application.CurrentFileItem().IsLiveTV())
```

Comment:
I know it's been like that before, but this IMO looks wrong in two ways:
a) when having a timeshift buffer, seeking to a time code within the buffer might still make sense
b) g_application.m_pPlayer->CanSeek() (which is checked in "onAction" method) should already take care of this